docs/configuration/trap-configuration.md

```diff
@@ -143,4 +143,10 @@ Example:
 ```yaml
 traps:
   aggregateTrapsEvents: "true"
+```
+
+### Updating trap configuration
+If you need to update some configuration regarding traps, you can do it by editing the `value.yaml` and then running below command to restart the pod deployment.
```

Comment:
`some configuration` doesn't sound well 🤔 
`values.yaml` instead of `values.yaml`

Reply:
fixed